packages/main/src/plugin/command-registry.ts

```diff
@@ -68,9 +66,8 @@ export class CommandRegistry {
     });
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async executeCommand<T = unknown>(commandId: string, ...args: any[]): Promise<T> {
-    const telemetryOptions = { commandId: commandId };
+  async executeCommand(commandId: string, ...args: unknown[]): Promise<unknown> {
```

Comment:
That may be break callers, no ? Or is the below change the only one ?

Reply:
this change is mostly internal so it's not breaking/impacting callers

Reply:
the signature in extension-api.d.ts (external view) is not changed